README.md

```diff
@@ -148,28 +153,78 @@ One may specify multiple data locations by a list of locations as long as they h
   ```
   location: ["./data/sample.parquet", "https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-01.parquet"]
   ```
-- `using`: string. Required.  
-One of `parquet`, `spark_sql` and `delta`, or the custom file format specified by the
-loader function in `steps/ingest.py` (ex. `csv`).
 
 
-- `loader_method`: string. Optional.  
-Method name of the custom loader function from `steps/ingest.py`.  
-<u>Example</u>: 
+</details>
+
+<details>
+<summary><strong><u>Using: "delta"</u></strong></summary>
+
+- `location`: string. Required.  
+Dataset locations or the catalog path
+<u>Examples</u>:
+  ```
+  location: catalog.schema.table
   ```
-  loader_method: load_file_as_dataframe
+  ```
+  location: "dbfs:/user/hive/warehouse/ml.db/all_prices_1990_2021_mar_csv"
```

Comment:
Can we update this example to not use `_csv` in the end? It could confuse people as they are csv files.